modelling/src/main/java/org/axonframework/modelling/command/inspection/ForwardingMessageHandlingMember.java

```diff
@@ -0,0 +1,21 @@
+package org.axonframework.modelling.command.inspection;
```

Comment:
I'm missing a copyright notice in this file.
You can copy-paste it from another file, or add the [copyright notice template](https://github.com/AxonFramework/AxonFramework/blob/master/copyright-template.xml) to your IDE.